### DISCUSSION TITLE
Ask about Data_Extraction_Visualization

### DISCUSSION THREAD

**Discussion by pxhien** [siddhartamukherjee/NEU-SURFACE-DEFECT#2] [General]
In 'Create_Validation_Images.ipynb', there is a command line: "from DATA_Extraction_Visualization import XMLExtraction" but  I didn't see any file named DATA_Extraction_Visualization. I guess this is the file from your previous project and it located in anacoda directory or somewhere but not on project's directory. Please tell me how to solve this problem?

**Comment by siddhartamukherjee** [comment]
That's true it was from a previous version. Not used in this notebook anymore. I will remove it in the next commit.
